src/sentry_plugins/amazon_sqs/plugin.py

```diff
@@ -159,14 +165,18 @@ def forward_event(self, event, payload):
                 logger.info(
                     metrics_name,
                     extra={
-                        "queue_url": queue_url,
-                        "access_key": access_key,
-                        "region": region,
+                        "test_number": 1,
                         "project_id": event.project.id,
                         "organization_id": event.project.organization_id,
-                        "message_group_id": message_group_id,
                     },
                 )
+                logger.info(
```

Comment:
@ceorourke If these aren't showing up in Datadog, I don't think it makes sense to test these

Reply:
Cool, double checked w/ Manu and they're not there - removed.